From feefa2b7520c69e8242f791196c8ef907ef1a26b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 10 Feb 2020 16:02:06 +0200
Subject: [PATCH] [SW-0] goya2_arc: pll: remove hl_pll_is_configurable()

ARC can configure the CPU PLL w/o any concern, especially when the CPU
is in reset.
There is no need for the Goya2 ARC private implementation of the
hl_pll_is_configurable() function.

Change-Id: Ifc3879e15af599f5150c8566a1da78295c415b8a
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya2_arc/asic.c b/goya2_arc/asic.c
index 7342ecb..5fb7a50 100644
--- a/goya2_arc/asic.c
+++ b/goya2_arc/asic.c
@@ -64,11 +64,6 @@
 	},
 };
 
-bool hl_pll_is_configurable(struct hl_pll_cfg *cfg)
-{
-	return cfg->base != mmCPU_PLL_BASE;
-}
-
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
 {
 	int pll_num = ARRAY_SIZE(goya_pll_cfg);
